04 - Guides, Workflows, & Courses/for Theme Designers.md

```diff
@@ -34,7 +34,7 @@ This note collects resources and guides for beginner and expert theme designers
 - [[CSS 101 - Community Talk]]
 
 ## Tools
-
+- [**Obsidian Theme Template**](https://github.com/obsidian-community/obsidian-theme-template) A template theme to get started
```

Comment:
Template theme or theme template?

Reply:
doesn't both essentially mean the same? 🤔

Reply:
I do think the order matters - but how about making it more explicit - such as "a template project to simplify starting a new theme"

Reply:
okay, I rephrased it.